src/_assets/css/_dash.scss

```diff
@@ -0,0 +1,163 @@
+$dash-callout: #121a26;
+$dash-sub-grey: #8d9399;
+$dash-dark-grey: #82878c;
+$dash-off-white: #f8f9fa;
+$dash-dark-black: #4a4a4a;
+$dash-highlight: #1967d2;
+$dash-off-black: #12202f;
+$dark-darker-black: #0d1520;
+
+.dash-header-callout {
+  background-color: $dash-callout;
+  text-align: center;
+  //todo: optimize
+  padding-top: 40px !important;
+  padding-bottom: 40px !important;
```

Comment:
does the todo mean to change this to `padding: 40px 0 40px 0 !important`?

Reply:
yes! cleaned up